.github/workflows/xts-collect-failed-run-logs.yaml

```diff
@@ -0,0 +1,115 @@
+        with:
+          egress-policy: audit
+
+      - name: Get run ID from run number
+        env:
+          GH_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+        id: run_id
+        run: |
+          RUN_ID=$(gh api -H "Accept: application/vnd.github+json" \
+            /repos/hashgraph/hedera-services/actions/workflows/zxcron-extended-test-suite.yaml/runs \
+            --jq '.workflow_runs[] | select(.run_number == ${{ inputs.workflow_id }}) | .id')
+          echo "::set-output name=value::$RUN_ID"
+
+      - name: Checkout Code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: '0'
+          ref: develop
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+
+      - name: Get run logs
+        env:
+          GH_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+        run: gh run view ${{ steps.run_id.outputs.value }} --log >> failed-run.log
+
+      - name: Upload log as artifact
+        id: upload-log
+        uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
+        with:
+          path: failed-run.log
+
+      - name: Report failure
+        if: ${{ !cancelled() && failure() && always() }}
+        uses: slackapi/slack-github-action@6c661ce58804a1a20f6dc5fbee7f0381b469e001 # v1.25.0
+        env:
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_CITR_WEBHOOK }}
+        with:
+          payload: |
+            {
+              "attachments": [
+                {
+                  "color": "#7647cd",
+                  "blocks": [
+                    {
+                      "type": "header",
+                      "text": {
+                        "type": "plain_text",
+                        "text": ":grey_exclamation: Hedera Services - New XTS logs available",
+                        "emoji": true
+                      }
+                    },
+                    {
+                      "type": "divider"
+                    },
+                    {
+                      "type": "section",
+                      "text": {
+                        "type": "mrkdwn",
+                        "text": "*Workflow ID*: ${{ inputs.workflow_id }}"
+                      }
+                    },
+                    {
+                      "type": "section",
+                      "text": {
+                        "type": "mrkdwn",
+                        "text": "*Workflow URL*: https://github.com/hashgraph/hedera-services/actions/runs/${{ steps.run_id.outputs.value }}"
+                      }
+                    },
+                    {
+                      "type": "section",
+                      "text": {
+                        "type": "mrkdwn",
+                        "text": "*Log file download URL*: ${{ steps.upload-log.outputs.artifact-url }}"
+                      }
+                    }
+                  ]
+                }
+              ]
+            }
```

Comment:
```suggestion
      - name: Share log information
         run: |
           echo "### Logs Collected for Workflow:" >> $GITHUB_STEP_SUMMARY
           echo "Workflow run ID: ${{ inputs.workflow_id  }}" >> $GITHUB_STEP_SUMMARY
           echo "Workflow URL: https://github.com/hashgraph/hedera-services/actions/runs/${{ steps.run_id.outputs.value }}" >> $GITHUB_STEP_SUMMARY
           echo "Log file download URL: ${{ steps.upload-log.outputs.artifact-url }}" >> $GITHUB_STEP_SUMMARY
```

Reply:
@rbarkerSL Are you suggesting to completely remove the Slack reporting step or only to add the "Share log information" step?

Reply:
I don't think we need the report step in this workflow.